front/pages/w/[wId]/members/index.tsx

```diff
@@ -129,343 +392,257 @@ export default function WorkspaceAdmin({
-                                            "relative cursor-default select-none py-1 pl-3 pr-9"
-                                          )
-                                        }
-                                        value={role}
-                                      >
-                                        {({ selected }) => (
-                                          <>
-                                            <span
-                                              className={classNames(
-                                                selected ? "font-semibold" : "",
-                                                "block truncate"
-                                              )}
-                                            >
-                                              {role}
-                                            </span>
-
-                                            {selected ? (
-                                              <span
-                                                className={classNames(
-                                                  "text-action-600",
-                                                  "absolute inset-y-0 right-0 flex items-center pr-4"
-                                                )}
-                                              >
-                                                <CheckIcon
-                                                  className="h-4 w-4"
-                                                  aria-hidden="true"
-                                                />
-                                              </span>
-                                            ) : null}
-                                          </>
-                                        )}
-                                      </Listbox.Option>
-                                    )
-                                  )}
-                                </Listbox.Options>
-                              </div>
-                            </>
-                          )}
-                        </Listbox>
-                      )}
-                    </div>
-                  </li>
                 ))}
-              </ul>
-            </div>
+              </DropdownMenu.Items>
+            </DropdownMenu>
           </div>
+          <Page.P>
+            The role defines the rights of a member of the workspace.{" "}
+            {roleTexts[member.workspaces[0].role]}
+          </Page.P>
+        </div>
+        <div className="flex flex-none flex-col gap-2">
+          <div className="flex-none">
+            <Button
+              variant="primaryWarning"
+              label="Revoke member access"
+              size="sm"
+              onClick={() => handleMemberRoleChange(member, "none")}
+            />
+          </div>
+          <Page.P>
+            Deleting a member will remove them from the workspace. They will be
+            able to rejoin if they have an invitation link.
```

Comment:
NIT: should we say "They will be able to rejoin if the invitation link is activated for a domain and they still have access to an email address for this domain." ? or too complex?